nle/env/base.py

```diff
@@ -131,6 +131,27 @@
 )
 
 
+# TODO: can probably be imported from somewhere?
+COLORS = [
```

Comment:
I think this should work with all ansi compatible terminals, but Im not entirely sure. It could probably be clearer which colour is which as well... But also this might not be a priority.